src/config/Typography.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/blocks';
+import { ThemeProvider, Typography } from '@mui/material';
+import theme from '@/config/theme';
+
+<Meta title='Typography' />
+
+export const SampleText =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis eu diam sed ipsum vulputate condimentum accumsan et massa. Pellentesque sit amet placerat purus. Vestibulum quis risus at lacus scelerisque hendrerit sed a ante. Nulla vel rhoncus arcu. Sed efficitur fringilla aliquam. Etiam sagittis bibendum urna, in imperdiet tellus commodo nec.';
+
+# Typography
```

Comment:
Interesting, I have never noticed this random border thats under h2s...
<img width="267" alt="Screenshot 2024-10-10 at 4 09 58 PM" src="https://github.com/user-attachments/assets/bfc32da5-15fb-47ef-afaf-76ba581d02c9">

Reply:
yeah! I think we never(?) use h1 or h2 anywhere in the application. The only places where I can find usages of these are component-only, so they use a different variant. E.g. `<Typography component='h2' variant='overline'>`. All part of why Bobby's [proposed changes to our type hierarchy](https://www.figma.com/design/kqZiUPjXvqSvJVslJCnlTB/OP-HMIS-%2F-2024-%2F-Q3-%2B-Q4-(Aug-1---Dec-31)?node-id=7616-1873&node-type=section&t=W2aYBQuRmyQ01UuX-0) make so much sense